web/locales/en/plugin__network-observability-plugin.json

```diff
@@ -1,13 +1,22 @@
 {
-  "Date & time": "Date & time",
-  "Src pod": "Src pod",
-  "Dst pod": "Dst pod",
-  "Src namespace": "Src namespace",
-  "Dst namespace": "Dst namespace",
-  "Src port": "Src port",
-  "Dst port": "Dst port",
-  "Protocol": "Protocol",
-  "Bytes": "Bytes",
-  "Packets": "Packets",
-  "Network Traffic": "Network Traffic"
-}
+    "Date & time": "Date & time",
+    "Src pod": "Src pod",
+    "Dst pod": "Dst pod",
+    "Src namespace": "Src namespace",
+    "Dst namespace": "Dst namespace",
+    "Src port": "Src port",
+    "Dst port": "Dst port",
+    "Protocol": "Protocol",
+    "Bytes": "Bytes",
+    "Packets": "Packets",
+    "Network Traffic": "Network Traffic",
+    "Refresh off": "Refresh off",
+    "{{count}} second": "{{count}} second",
+    "{{count}} second_plural": "{{count}} seconds",
```

Comment:
Maybe the translation team will complain about this `_plural` key (it happened to me recently). I'd just let it as: `"{{count}} seconds": "{{count}} seconds"`, if possible (same for the rest of times).

Reply:
I agree, However, this directly come from console [public translation file](https://github.com/openshift/console/blob/824dc2843e5072dc2d70ff4bfdb72c9db480234f/frontend/public/locales/en/public.json#L89) 

We have multiple solutions here: 
- Reuse their translations without duplicate (but with a risk if they rename the key)
- Copy / past as I did
- Do our own key / values as you suggest

I didn't reused their file since all the keys are sentences in English. I guess they can change key easily.